.circleci/config.yml

```diff
@@ -0,0 +1,76 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/coala
+
+    docker:
+      - image: circleci/python:3.6.1
+
+    parallelism: 2
```

Comment:
nice.  delete our hacks in the `.misc` scripts which relate to number of jobs.  `git grep -i circle`